packages/front-end/components/Layout/SidebarLink.module.scss

```
@@ -39,7 +39,7 @@ $topbarheight: 60px;
   }
 
   &:hover {
-    background-color: scale-color($dark-purple, $lightness: +12%);
+    background-color: color-mix(in srgb, var(--violet-9) 30%, transparent);
```

Doesn't look like `color-mix` is super widely supported yet across browsers.  Does it fall back gracefully?

---

what is our oldest browser support that we want? the only other thing we can do is fall back to a hardcoded color (not from radix)